dids/src/main/kotlin/web5/sdk/dids/ion/model/Models.kt

```diff
@@ -163,6 +233,38 @@ public data class SidetreeCreateOperation(
 
 }
 
+/**
+ * Sidetree update operation as defined in https://identity.foundation/sidetree/api/#update
+ */
+public data class SidetreeUpdateOperation(
+  public val type: String,
+  public val didSuffix: String,
+  public val revealValue: Reveal,
+  public val delta: Delta,
+  public val signedData: String,
+)
+
+/**
+ * Sidetree recover operation as defined in https://identity.foundation/sidetree/api/#recover
+ */
+public data class SidetreeRecoverOperation(
+  public val type: String,
+  public val didSuffix: String,
+  public val revealValue: Reveal,
+  public val delta: Delta,
+  public val signedData: String,
+)
+
+/**
+ * Update operation signed data object as defined in https://identity.foundation/sidetree/spec/#update-signed-data-object
+ */
+public data class UpdateOperationSignedData(
+  @JsonSerialize(using = JacksonJwk.Serializer::class)
+  @JsonDeserialize(using = JacksonJwk.Deserializer::class)
```

Comment:
You can create a Jackson module that is registered on the ObjectMapper once, instead of annotating every use of a type
https://github.com/TBD54566975/tbdex-kt/blob/main/protocol/src/main/kotlin/tbdex/sdk/protocol/serialization/Json.kt#L35

Reply:
Thanks! I'm torn. On one hand, it's nice to not have to annotate all usages, but on the other hand, allowing developers to use this class with their own mapper without the need to register a custom module is also nice.

I'd like to keep as is, just because the devex experience is nicer. Happy to revisit!